DavidVelaz17/ConsultorioMedico
# IWAID-24 Migración de BD con Flyway

### Ticket:
* [IWAID-24](https://iwaids.atlassian.net/browse/IWAID-24?atlOrigin=eyJpIjoiYmU2ZWYyNmIyM2RjNGE2YzkxMmZjOThjZGMwOWYyODAiLCJwIjoiaiJ9)

### What Changed:
* Flyway implemented
* Created Doctor's and Medicine's migration

### How To Test:
* Fist you must change the username and password to yours in the application.properties file, it's located under the resources folder
* Then delete the consultingroomiwa schema from your MySQL Workbench (if exists, if not go to the next step), to do this, right click on consultingroomiwa scheema and select "Drop Schema..."
* Create a new schema with the same name (consultingroomiwa), to do this, select this icon ![image](https://github.com/DavidVelaz17/iwaid-api/assets/139006505/0803d589-06da-4cc9-befb-dc357b68236d) from the top of MySQL Workbench, in the oppened window copy: "consultingroomiwa", and paste it in the "Name:" section. 
  * Like this:   
![image](https://github.com/DavidVelaz17/iwaid-api/assets/139006505/72a83ee7-ee42-47e6-afb9-af78f9a50e10)
* To finish creating the schema click the "Apply" button on the bottom of the window.
* Then run the project, now you can refresh the schemas seccion on you MySQL Workbench to see the changes

### Evidence:
* Before:
![image](https://github.com/DavidVelaz17/iwaid-api/assets/139006505/eb82c779-438b-4830-8a8b-c45f40e838e5)
* After:
![image](https://github.com/DavidVelaz17/iwaid-api/assets/139006505/b59e6c28-cc31-41c2-8a4d-5c2e7ab572c4)

* doctors table:
![image](https://github.com/DavidVelaz17/iwaid-api/assets/139006505/9eeb5f52-170d-40a7-964a-38ec5bb44296)
* medicines table:
![image](https://github.com/DavidVelaz17/iwaid-api/assets/139006505/d52e0321-35c3-415a-b777-d9b2c2310d3b)
